kubernetes/src/main/java/io/kubernetes/client/ApiClient.java

```diff
@@ -768,10 +768,10 @@ public String selectHeaderAccept(String[] accepts) {
      *
      * @param contentTypes The Content-Type array to select from
      * @return The Content-Type header to use. If the given array is empty,
-     *   JSON will be used.
+     *    or matches any, JSON will be used.
      */
     public String selectHeaderContentType(String[] contentTypes) {
-        if (contentTypes.length == 0) {
```

Comment:
Sounds good. 

And thanks - I've done one myself now too so this shouldn't be too bad.